### PR TITLE
TWEAK: Stop get_comment_time() if null

### PR DESCRIPTION
**Pull request by DSGND**
Check if function get_comment_id is null before executing it.

Trac ticket: https://core.trac.wordpress.org/ticket/52322

### CODE REVIEW

File: src/wp-includes/comment-template.php [hunk 1]
```diff
@@ -1034,15 +1034,21 @@ function comment_text( $comment_ID = 0, $args = array() ) {
  * Retrieves the comment time of the current comment.
  *
  * @since 1.5.0
+ * @since 4.4.0 Added the ability for `$comment_ID` to also accept a WP_Comment object.
```
Comment (audrasjb) [inline src/wp-includes/comment-template.php]: Replace `4.4.0` with `6.2.0` :)

File: src/wp-includes/comment-template.php [hunk 2]
```diff
@@ -1034,15 +1034,21 @@ function comment_text( $comment_ID = 0, $args = array() ) {
...
+ *                                   Default true.
+ * @param int|WP_Comment $comment_ID WP_Comment or ID of the comment for which to get the date.
  * @return string The formatted time.
  */
-function get_comment_time( $format = '', $gmt = false, $translate = true ) {
-	$comment = get_comment();
+function get_comment_time( $format = '', $gmt = false, $translate = true, $comment_ID = 0 ) {
+	$comment = get_comment( $comment_ID );
+
+	if ( null === $comment ) {
```
Comment (audrasjb) [inline src/wp-includes/comment-template.php:1050]: I don't think we really need this check, isn't it?
It doesn't exist in `get_comment_date()`.

**Review by audrasjb — CHANGES_REQUESTED** [review]
Thanks for the PR! Two feedback added.

File: src/wp-includes/comment-template.php [hunk 3]
```diff
@@ -1034,15 +1034,21 @@ function comment_text( $comment_ID = 0, $args = array() ) {
...
  *
- * @param string $format    Optional. PHP time format. Defaults to the 'time_format' option.
- * @param bool   $gmt       Optional. Whether to use the GMT date. Default false.
- * @param bool   $translate Optional. Whether to translate the time (for use in feeds).
- *                          Default true.
+ * @param string         $format     Optional. PHP date format. Defaults to the 'time_format' option.
+ * @param bool           $gmt        Optional. Whether to use the GMT date. Default false.
+ * @param bool           $translate  Optional. Whether to translate the time (for use in feeds).
+ *                                   Default true.
+ * @param int|WP_Comment $comment_ID WP_Comment or ID of the comment for which to get the date.
```
Comment (costdev) [inline src/wp-includes/comment-template.php]:
```suggestion
 * @param int|WP_Comment $comment_ID Optional. WP_Comment or ID of the comment for which to get the date.
                                     Default is 0, or the global comment.
```
Just a quick note that this parameter is optional and should be documented as such. 🙂

Comment (mukeshpanchal27) [reply src/wp-includes/comment-template.php]: @DSGND ☝️

**Comment by audrasjb** [comment]
I added one commit to the PR to fix a WPCS issue

**Review by costdev — APPROVED** [review]

**Review by audrasjb — APPROVED** [review]
lgtm

**Comment by audrasjb** [comment]
committed in https://core.trac.wordpress.org/changeset/55284

**Pull request closed without merging**